docroot/WEB-INF/src/edu/osu/cws/evals/tests/BackendMgrTests.java

```diff
@@ -326,6 +326,43 @@ public void shouldSendProfFacultyEmailsRightAwayOnceReviewPeriodStarts() throws
         assert !BackendMgr.timeToSendFirstStatusEmail(appraisal, new DateTime());
     }
 
+    public void shouldUpdateSalaryWhenAppraisalIsDue() throws Exception {
```

Comment:
this commit defines a new method but does not call it. i assume `shouldUpdateSalaryWhenAppraisalIsDue()` is called elsewhere.